### DISCUSSION TITLE
For those want to install foundry in a NixOS machine

### DISCUSSION THREAD

**Discussion by Reblixt** [foundry-rs/foundry#7236] [Q&A]
I had trouble to install foundry on my NixOS pc for about a week and finally stumbled on a solution. 

If you want to install foundry in a NixOS pc.
This "fix" allows your NixOS to execute normal binarys.
Add this in your configuration.nix file.
```
programs.nix-ld.enable = true;
programs.nix-ld.libraries = with pkgs; [   
  glibc
  gcc.cc.lib 
  ];
  ```
  Then reboot
  
  Then initiate "foundryup" again.

**Comment by prettymuchbryce** [comment]
Any chance we can get an official foundry flake for nix users?
